package-lock.json

```diff
@@ -2425,7 +2424,7 @@
         },
         "event-stream": {
             "version": "3.3.4",
-            "resolved": "http://registry.npmjs.org/event-stream/-/event-stream-3.3.4.tgz",
+            "resolved": "https://registry.npmjs.org/event-stream/-/event-stream-3.3.4.tgz",
```

Comment:
Oh since it's been long since I contributed to this repo, I ran `npm install` again and it updates the `package-lock.json`. Lemme know if I should revert this. Thanks

Reply:
It should be ok but, to be safe, it would be best to back this one out. Thanks @kenrick95

Reply:
okay, reverted :)